src/App.js

```diff
@@ -14,8 +15,12 @@ function App() {
 				<Routes>
 					<Route path="/" element={<Home />}></Route>
 					<Route path="/articles" element={<Articles />}></Route>
-          <Route
-						path="/api/articles/:topic"
+					<Route
+						path="/api/articles/:article_id"
+						element={<SingleArticleCard />}
+					></Route>
+					<Route
+						path="/api/articles/topic/:topic"
```

Comment:
should these paths be /api? 
I might suggest:
/articles/:article_id
/topics/:topic

Reply:
Sure, makes more sense to have them without /api
Will change them.